city_scrapers/spiders/wicks_sccab.py

```diff
@@ -0,0 +1,86 @@
+import re
+from urllib.parse import urljoin
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class WicksSccabSpider(CityScrapersSpider):
+    name = "wicks_sccab"
+    agency = "Sedgwick County: Corrections Advisory Boards"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.sedgwickcounty.org/corrections/corrections-advisory-boards/"
+    ]
+
+    def parse(self, response):
+        """Parse meeting items from agency website."""
+        start_time = "8 a.m."
+        end_time = "9 a.m."
+        location = {
+            "name": "Adult Field Services",
+            "address": "905 N. Main, Wichita, Kansas",
+        }
+        for item in response.css("h4:nth-of-type(2) + *").css("li"):
+            date = re.search(r"\D+ \d+, \d+", item.css("::text").get()).group()
+            meeting = Meeting(
+                title="Community Corrections Advisory Board Monthly Meeting",
+                description=self._parse_description(item),
+                classification=BOARD,
+                start=self._parse_start(date, start_time),
+                end=self._parse_end(date, end_time),
+                all_day=False,
+                time_notes="",
+                location=location,
+                links=self._parse_links(response, item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_description(self, item):
+        """
+        Parse description.
+        Used to indicate if meeting has been cancelled.
+        Helps _get_status function set status correctly.
+        """
+        text = item.get().lower()
+        return "CANCELLED" if "cancel" in text else ""
```

Comment:
**praise:** Good stuff! Thanks for giving some consideration to how `_get_status` detects cancellations.